src/main/java/at/hannibal2/skyhanni/mixins/transformers/renderer/MixinRendererLivingEntityHookSBA.java

```diff
@@ -15,19 +12,11 @@
 @Mixin(targets = "codes/biscuit/skyblockaddons/asm/hooks/RendererLivingEntityHook")
 public class MixinRendererLivingEntityHookSBA<T extends EntityLivingBase> {
 
-    @Unique
-    private static final RendererLivingEntityHook skyHanni$hook = new RendererLivingEntityHook();
-
     @Inject(method = "equals", at = @At("HEAD"), cancellable = true, remap = false)
     private static void onEquals(String displayName, Object otherString, CallbackInfoReturnable<Boolean> cir) {
-        if (skyHanni$hook.isCoolPerson(displayName)) {
+        if (RendererLivingEntityHook.shouldBeUpsideDown(displayName)) {
             cir.setReturnValue(true);
         }
     }
 
-    @Inject(method = "isWearing", at = @At("HEAD"), cancellable = true, remap = false)
-    private static void onIsWearing(EntityPlayer player, EnumPlayerModelParts p_175148_1_, CallbackInfoReturnable<Boolean> cir) {
```

Comment:
Isnt this still needed? as SBA used a transformer to replace the vanilla isWearing with this method

Reply:
Oh yeah, looks like SBA is just straight up replacing the call instead of just putting in an `||`.